python-backend/app/api/mines/permits/permit_amendment/resources/permit_amendment.py

```diff
@@ -23,28 +23,20 @@ class PermitAmendmentListResource(Resource, UserMixin):
         help='GUID of the party that is the permittee for this permit.',
         location='json')
     parser.add_argument(
-        'received_date',
-        location='json',
-        type=lambda x: datetime.strptime(x, '%Y-%m-%d') if x else None,
-        store_missing=False)
+        'received_date', location='json', type=inputs.datetime_from_iso8601, store_missing=False)
     parser.add_argument(
-        'issue_date',
-        location='json',
-        type=lambda x: datetime.strptime(x, '%Y-%m-%d') if x else None,
-        store_missing=False)
+        'issue_date', location='json', type=inputs.datetime_from_iso8601, store_missing=False)
     parser.add_argument(
         'authorization_end_date',
         location='json',
-        type=lambda x: datetime.strptime(x, '%Y-%m-%d') if x else None,
+        type=inputs.datetime_from_iso8601,
```

Comment:
Is this going to require a change on the frontend?

Reply:
It did not, i used this method in the permit amendment put and tested it. and it worked with no issue. Showed in the form and was parsed correctly in he backend.